metrics/next-android-render-test-runner-probe-memory.json

```diff
@@ -1,11 +0,0 @@
-{
-    "base_test_path":"mapbox-gl-js/test/integration",
-    "expectation_paths":["render-test/expected", "render-test/expected/android"],
-    "ignore_paths":["render-test/ignores/ignores.json", "render-test/ignores/linux-ignores.json", "render-test/ignores/android-temp-fail.json"],
-    "vendor_path":"vendor",
-    "asset_path": "mapbox-gl-js/test/integration",
-    "result_path":"/sdcard/",
-    "probes": ["probeMemory"],
-    "metric_path":"metrics/next-android-render-test-runner",
-    "filter": "real-world"
```

Comment:
This removes memory metrics for real-world styles, would we add new memory metrics test-suite in follow-up PRs?

Reply:
Yes, that is the idea. I will add them back for all the platforms with metrics.